packages/twenty-front/src/modules/object-record/hooks/useObjectRecordBoard.ts

```diff
@@ -43,7 +43,7 @@ export const useObjectRecordBoard = () => {
     savedPipelineStepsState,
   );
 
-  const filter = turnFiltersIntoObjectRecordFilters(
+  const filter = turnUIFiltersIntoObjectRecordFilters(
```

Comment:
I don't think it is the right naming but I am not sure about what it should be:
turnObjectDropdownFilterIntoQueryFilter?